CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**JDBC Driver 3.13.17**
+
+- \SNOW-571607 | | Create patch for driver release v3.13.16 to fix incorrect behavior for getSchemas() function 
```

Comment:
Can we add the other 2 release notes?
-Fix setting of invalid JVM parameters proxyHost and proxyPort
-SNOW-348660: Add getters for the timezone and ZonedDateTime for SnowflakeTimestampWithTimezone object